device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/sesto-1.json

```diff
@@ -0,0 +1,256 @@
+{
+  "lanes": [
+    {
+      "index": 200,
+      "local_lane_id": 0,
+      "system_side": true,
+      "tx_polarity": 0,
+      "rx_polarity": 0,
+      "line_tx_lanemap": 0,
+      "line_rx_lanemap": 0,
+      "line_to_system_lanemap": 0,
+      "mdio_addr": "0x0200"   
```

Comment:
i thought the mdio_addr is per phy, or per core, and it seems you have unique mdio_addr per lane. am i missing something here?

Reply:
We are not currently using this mdio_addr attribute. This attribute was recommended in the original SAI_Gearbox_API_Proposal documentation.